code/src/cljs/sixsq/nuvla/ui/edge/views.cljs

```diff
@@ -193,9 +196,33 @@
         [:<>
          [uix/ModalHeader {:header (str nuvlabox-name-or-id " created") :icon "box"}]
 
-         (when @new-private-ssh-key
+         (when (or @new-private-ssh-key playbooks-toggle)
            [ui/Segment {:basic true}
-            [CreateSSHKeyMessage @new-private-ssh-key private-ssh-key-file tr]])
+            (when playbooks-toggle
+              [ui/Message {:icon true}
+               [ui/Icon {:name (if @playbooks-cronjob "check circle outline" "spinner")
+                         :loading (if @playbooks-cronjob false true)}]
+               [ui/MessageContent
+                [ui/MessageHeader [:span (@tr [:nuvlabox-playbooks-cronjob]) " "
+                                   (when @playbooks-cronjob
+                                     [ui/Popup {:content @playbooks-cronjob
```

Comment:
Now that I merge this [PR](https://github.com/nuvla/api-server/commit/c69c9e3344aba25eecd27ad599d75379f9b5079c) this PR need to be fixed